console.go

```diff
@@ -27,6 +22,9 @@ const (
 	colorMagenta
 	colorCyan
 	colorWhite
+
+	colorBold     = 1
+	colorDarkGray = 90
```

Comment:
I've moved these into the same `const` block because the only thing they have in common with each other is that they don't fit tidily into the iota sequence.  We could pull them back out if that's preferred though.